### ISSUE TITLE
Real elytra texture for the reinforced elytra (and more?).

### ISSUE THREAD

**Issue by oddlama** [oddlama/vane#125]
This can work by supplying a special file for optifine clients in the resource pack. I can do the coding, but I don't know if I can create a convincing texture.

If we are going there anyway I'd also be willing to include some skins for the elytra (craftable? selectable?). Not sure which ones though (art is hard for me).

**Comment by ryantheleach** [comment]
Is the reason this would need to be optifine only, is that customModelData doesn't apply to armor?

I only just found this out, from this issue, and it seems a shame.

I wonder if it was an oversight, or just that CustomModelData was only supported for items so far.

**Comment by oddlama** [comment]
Yes customModelData seems to be only supported for items. And because armor textures are not items i guess they are not affected. A real shame, but optifine adds the ability to select by item name, which would inherently work in this case.

(Related note: Could also be used to add copper armor or something like that, but not sure I want it.)

**Comment by oddlama** [comment]
Not a "bug" per-se, but blocked on mojang allowing us to select equipment textures based on customModelData.

**Comment by ryantheleach** [comment]
If you ever revisit this, I think elytra in a loom would fit for design crafting.

**Comment by oddlama** [comment]
I'll definitely revisit it once I find a way to make it work :D
The loom definitely sounds perfect for this!
